Enabling Third-party Smart Contract Account Applications on Polkadot Based on Account Abstraction.md

```diff
@@ -0,0 +1,154 @@
+- **Estimated duration:** 2 months
+- **FTE:**  3
+- **Costs:** 10,000 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License | Apache 2.0 |
+| **0b.** | Documentation | We will provide both **inline documentation** of the code and a basic **tutorial** that explains how a developer can develop his/her third-party account applications via our standard. |
+| **0c.** | Testing and Testing Guide | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests to verify our AA standard can fully support third-party account applications. |
+| **0d.** | Docker | We will provide Dockerfiles that can be used to test all the functionality delivered with this milestone. |
+| 0e. | Article | We will publish an article that explains how our AA standard works and runs for developers to smoothly provide users with different account applications in their AA accounts. |
+| 1. | User interface | We will design and develop a sample AA wallet for users to have their AA account and equip it with third-party account applications. |
+| 2. | Smart contracts | We will deliver a set of ink! smart contracts to show the specifications of our AA standard supporting third-party account applications. |
+
+
+### Milestone 2 Example — Use Cases of Account applications
+
+- **Estimated duration:** 2 months
+- **FTE:**  3
+- **Costs:** 9,000 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License | Apache 2.0 |
+| **0b.** | Documentation | We will provide both **inline documentation** of the code and a basic **tutorial** that explains how users can use our use cases for account management and interaction applications. |
+| **0c.** | Testing and Testing Guide | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests to verify the functionalities and security of our use cases.  |
+| **0d.** | Docker | We will provide Dockerfiles that can be used to test all the functionality delivered with this milestone. |
+| 0e. | Article | We will publish an article that explains how our use cases work and run based on our AA standard to encourage more developers to develop their third-party account applications. |
+| 1. | User interface | We will design and develop two use cases for users to use third-party account applications via our AA standard. One is decentralized universal login as an account management application, and the other one is a decentralized automatic trading bot as an account interaction application. |
+| 2. | Smart contracts | We will deliver a set of ink! smart contracts to show the specifications of our third-party account applications as the use cases of our AA standard supporting any third-party account applications. |
+
+
+### Milestone 2 Example — Demand Abstraction
```

Comment:
```suggestion
### Milestone 3 — Demand Abstraction
```

Reply:
Thanks for your critiques and we have revised them!